yb-voyager/src/srcdb/source.go

```diff
@@ -315,6 +336,93 @@ func (source *Source) extrapolateDSNfromSSLParams(DSN string) string {
 	return DSN
 }
 
+func (source *Source) getPgDumpSchemaArgsFromFile() string {
```

Comment:
Don't think this method should be on the source struct. (Remember we were discussing this?). It doesn't use the source struct's fields/methods. Moreover, this is pg specific functionality that shouldn't be on the abstract `source` struct. Move it to another file `pg_dump.go` if it's common to both pg_dump schema and data. 

Speaking of, I made some related refactoring in https://github.com/yugabyte/yb-voyager/pull/929 😄  Do take a look !